src/validators/url.py

```diff
@@ -108,16 +103,21 @@ def _validate_netloc(
     ) and _validate_auth_segment(basic_auth)
 
 
-def _validate_optionals(path: str, query: str, fragment: str):
+def _validate_optionals(
+    path: str,
+    query: str,
+    fragment: str,
+    strict_query: bool = False,
```

Comment:
A small change:

```diff
- strict_query: bool = False,
+ strict_query: bool,
```

You can make it a required parameter, since the default value is already set at `def url(...)`.